tools/st2-generate-symmetric-crypto-key.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+
+import argparse
+import os
+import sys
+
+from keyczar.keys import AesKey
+
+
+def main(key_path, force=False):
+    base_path = os.path.dirname(key_path)
+    if not os.access(base_path, os.W_OK):
+        print('ERROR: You do not have sufficient permissions to write to path: %s.' % key_path)
+        print('Try setting up permissions correctly and then run this tool.')
+        sys.exit(1)
+
+    if os.path.exists(key_path):
+        print('You already have a key at the specified location %s!' % key_path)
+
+        if not force:
+            print('Not generating a new key. Either delete the file or re-run with --force.')
+            sys.exit(2)
+
+        print('WARNING: Rewriting existing key with new key!')
+
+    with open(key_path, 'w') as key_file:
+        k = AesKey.Generate()
+        key_file.write(str(k))
```

Comment:
If someone prefers keyczar they are free use keyczar, we have a wrapper that makes things a little bit easier and is our opinion. If we need to add more features to this script making it as complex as keyczar we can abandon this approach. At this point it certainly adds value by making things simpler for a user.